src/wazuh_qa_framework/system/wazuh_handler.py

```diff
@@ -843,3 +844,104 @@ def is_manager(self, host):
             bool: True if host is manager
         """
         return host in self.get_managers()
+
+    def create_group(self, group_name):
+        """Function to create a group.
+        Args:
+            group_name (str): Name of the group.
+        """
+        self.logger.info(f'creating group {group_name}')
+        self.run_command('manager1', f"/var/ossec/bin/agent_groups -q -a -g {group_name}", become=True)
```

Comment:
I would add the manager as a parameter

```suggestion
    def create_group(self, manager, group_name):
        """Function to create a group.
        Args:
            group_name (str): Name of the group.
        """
        self.logger.info(f'creating new {manager} group: {group_name}')
        self.run_command(manager, f"/var/ossec/bin/agent_groups -q -a -g {group_name}", become=True)
```

Reply:
Done in https://github.com/wazuh/qa-system-framework/pull/71/commits/6e4fa8a41bfb8248e339d6eb0911ce229ca57084